sections/sprint-1/networking.tex

```diff
@@ -263,3 +263,62 @@ \subsection{IPv4 and IPv6}\label{ipv4-ipv6}
 IPv6 addresses are usually written as eight 16-bit hexadecimal numbers \cite{socketnetworking}.
 The higher-order bits of the 128-address imply the type.
 For this project, this should not have a significant impact, and either can be used.
+
+
+\subsection{Format of data}
+As it is needed to send data over the network for our game, it is required to define the format it is sent as.
+The network format for the field anchor, player position and goal position is \texttt{0xyyyyxxxxiisstt}, where \texttt{t} is the type, \texttt{s} is the timestamp, \texttt{i} is the ID, \texttt{xxxx} is the \texttt{x} position and \texttt{yyyy} is the \texttt{y} position.
+These formats can be seen on \autoref{Tab:anchor-format}, \autoref{Tab:player-format} and \autoref{Tab:player-format}.
+These are almost identical, as the same information is required.
+The only difference between them is the type, and the context of the IDs.
+The type is at the end of the hexadecimal, so that it is easy to bitshift, and then switch on which type it is.
+After the type the timestamp is placed, which is used to detect outdated data.
+\\\\
+It was decided to use 2 and 4 hexadecimal, so that it is in byte format and hereby straightforward to work with in \texttt{C\#}.
```

Comment:
```suggestion
It was decided to use a size of respectively 2 and 4 hexadecimal so that it is in byte format and i straightforward to work with in the \texttt{C\#} language.
```

er det her det fulde argument? @jenspetur94

Reply:
Det fulde argument er at 2 og 4 hexadecimaler svarer til 1 og 2 bytes. Det er nemmere at arbejde med i C# fordi at enkelte dele af package har samme størrelse som nogle type i C#. Derfor kan man bitshifte sådan at det argument i pakken man skal aflæse forrest og så type caste packagen til den type der svarer til størrelsen på argumentet.

Reply:
![image](https://user-images.githubusercontent.com/22811843/77744661-35a9c300-701a-11ea-8d20-86c2dad46b34.png)